### PR TITLE
quazip 1.1

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

followup #63781

### CODE REVIEW

**Comment by chenrui333** [comment]
```
In file included from test.cpp:1:
In file included from /usr/local/Cellar/quazip/1.1/include/QuaZip-Qt5-1.1/quazip/quazipfile.h:28:
In file included from /usr/local/Cellar/qt/5.15.2/lib/QtCore.framework/Headers/QIODevice:1:
In file included from /usr/local/Cellar/qt/5.15.2/lib/QtCore.framework/Headers/qiodevice.h:45:
In file included from /usr/local/Cellar/qt/5.15.2/lib/QtCore.framework/Headers/qobject.h:49:
In file included from /usr/local/Cellar/qt/5.15.2/lib/QtCore.framework/Headers/qlist.h:47:
In file included from /usr/local/Cellar/qt/5.15.2/lib/QtCore.framework/Headers/qhashfunctions.h:47:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/numeric:148:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/cmath:324:7: error: no member named 'isgreaterequal' in the global namespace; did you mean '::std::greater_equal'?
using ::isgreaterequal;
      ^~
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:767:29: note: '::std::greater_equal' declared here
struct _LIBCPP_TEMPLATE_VIS greater_equal : binary_function<_Tp, _Tp, bool>
```

**Comment by chenrui333** [comment]
See discussions in [here](https://forums.swift.org/t/is-anyone-else-getting-this-error-when-building-the-compiler-from-master-on-macos/36113/10), this might be related to swift upgrade

**Comment by carlocab** [comment]
@erinacio proposed a slightly different test, maybe it'll work: https://github.com/Homebrew/brew/issues/10152#issuecomment-752211655

The build instructions are also slightly different, but I don't see how much of a difference it would make...

**Comment by chenrui333** [comment]
as upstream reports the ARM support, add the label to reflect that.

**Comment by chenrui333** [comment]
add @erinacio's change into here.

**Comment by erinacio** [comment]
> add @erinacio's change into here.

Well...... As discussed in https://github.com/Homebrew/brew/issues/10152#issuecomment-752222018，your formula actually works with CLT only or Xcode only, but not with Xcode's compiler + CLT's macOS SDK, which seems to be the case of CI environment. No need to pick my changes. It doesn't pass `brew audit` and will still break CI.

File: Formula/quazip.rb [hunk 1]
```diff
@@ -12,13 +14,15 @@ class Quazip < Formula
...
   depends_on xcode: :build
+
   depends_on "qt"
 
   def install
-    system "qmake", "quazip.pro", "-config", "release",
-                    "PREFIX=#{prefix}", "LIBS+=-lz"
-    system "make", "install"
+    system "cmake", "-DQUAZIP_DIR_NAME=", ".", *std_cmake_args
+    system "cmake", "--build", "."
```
Comment (jonaski) [inline Formula/quazip.rb:24]:
```suggestion
    system "make", "CPATH="
```
Should fix the cmath header issues.

**Comment by chenrui333** [comment]
This might be something that I can use with other PRs as well. qxmpp 1.3.2 #68625

**Pull request closed without merging**